dspace-uclouvain/src/main/java/org/dspace/core/model/DialPerson.java

```diff
@@ -49,43 +68,43 @@ public String getEmployeeId(){
     }
 
     public String getAlternateName() {
-        return this.alternatename;
+        return this.alternateName;
     }
 
     public void setAlternateName(String alternateName) {
-        this.alternatename = alternateName;
+        this.alternateName = alternateName;
     }
 
     public String getFirstname() {
-        return this.firstname;
+        return this.firstName;
     }
 
-    public void setFirstname(String firstname) {
-        this.firstname = firstname;
+    public void setFirstname(String firstName) {
+        this.firstName = firstName;
     }
 
     public String getId_patron() {
-        return this.id_patron;
+        return this.idPatron;
     }
 
-    public void setId_patron(String id_patron) {
-        this.id_patron = id_patron;
+    public void setId_patron(String idPatron) {
+        this.idPatron = idPatron;
     }
 
-    public String getLastname() {
```

Comment:
Cosmetic : 
You loose ! Try again :-)
`Lastname` is an existing word in english --> `getLastname`; `getFirstname` too.
`getAlternateName` because "alternate" and "name" are two separated words (that could be merged using '-' but two words!)